src/utils/getCirclePoints.ts

```diff
@@ -0,0 +1,53 @@
+import { circle } from 'bresenham-zingl';
+
+import { Point } from '../geometry';
+
+/**
+ * Get the coordinates of the points on a circle. The reference is the center of the circle.
+ * The first point is the right one and they are then sorted clockwise.
+ *
+ * @param radius - Radius of the circle.
+ * @returns The coordinates of the points on a circle of given diameter.
+ */
+export function getCirclePoints(radius: number): Point[] {
+  let circlePoints: Point[] = [];
+
+  circle(radius, radius, radius, (column: number, row: number) => {
+    circlePoints.push({ row: row - radius, column: column - radius });
+  });
+
+  let firstQuarter: Point[] = [];
+  let secondQuarter: Point[] = [];
+  let thirdQuarter: Point[] = [];
+  let fourthQuarter: Point[] = [];
+
+  for (let i = 0; i < circlePoints.length / 4; i++) {
```

Comment:
is `circlePoints.length` guaranteed to be a multiple of 4?

Reply:
You could also do `for (let i = 0; i < circlePoints.length; i+=4) {` to avoid having to multiply by 4 in the body of the loop.

Reply:
> is `circlePoints.length` guaranteed to be a multiple of 4?

I think it should be, the algorithm generating the circle points creates the points of a quarter of circle and then uses symmetries to get the three remaining quarters.